### PR TITLE
[PLA-13068] Drop citrine.builders.

### PR DESCRIPTION
**Pull request by anoto-moniz**
# Citrine Python PR

## Description 
It is a little used package, largely unmaintained, and one not many people know exists. Anything useful in it almost certainly belongs in citrinade instead.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Maintenance (non-breaking change to assist developers)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in __version\__.py

### CODE REVIEW

**Comment by anoto-moniz** [comment]
nextgen IT runs:
- development-e2e: https://jenkins.corp.citrine.io/job/nextgen-ITs/43193/
- stage: https://jenkins.corp.citrine.io/job/nextgen-ITs/43194/

File: tests/informatics/test_design_spaces.py [hunk 1]
```diff
@@ -124,14 +123,17 @@ def test_hierarchical_initialization(hierarchical_design_space):
 def test_data_source_build(valid_data_source_design_space_dict):
     ds = DesignSpace.build(valid_data_source_design_space_dict)
     assert ds.name == valid_data_source_design_space_dict["data"]["instance"]["name"]
+    assert ds.description == valid_data_source_design_space_dict["data"]["description"]
     assert ds.data_source == DataSource.build(valid_data_source_design_space_dict["data"]["instance"]["data_source"])
     assert str(ds) == f'<DataSourceDesignSpace \'{ds.name}\'>'
```
Comment (kroenlein) [inline tests/informatics/test_design_spaces.py:128]:
```suggestion
    assert str(ds) == f"<DataSourceDesignSpace '{ds.name}'>"
```
As long as we're here.

**Review by kroenlein — APPROVED** [review]
Looks good, assuming tests pass.

File: tests/informatics/test_design_spaces.py (hunk 1 above)
Comment (anoto-moniz) [reply tests/informatics/test_design_spaces.py:128]: Sure.

**Comment by kroenlein** [comment]
Note: with removal of src/citrine/builders/design_spaces.py, you've removed all citrine-python dependence on pandas.

**Pull request merged**